akka-stream/src/main/scala/akka/stream/javadsl/Graph.scala

```diff
@@ -99,6 +99,56 @@ object MergePreferred {
 }
 
 /**
+ * Merge several streams, taking elements as they arrive from input streams
+ * (picking from prioritized once when several have elements ready).
+ *
+ * A `MergePrioritized` has one `out` port, one or more input port with their priorities.
+ *
+ * '''Emits when''' one of the inputs has an element available, preferring
+ * a input based on its priority if multiple have elements available
+ *
+ * '''Backpressures when''' downstream backpressures
+ *
+ * '''Completes when''' all upstreams complete (eagerComplete=false) or one upstream completes (eagerComplete=true), default value is `false`
+ *
+ * '''Cancels when''' downstream cancels
+ *
+ * A `Broadcast` has one `in` port and 2 or more `out` ports.
+ */
+object MergePrioritized {
+  /**
+   * Create a new `MergePrioritized` stage with the specified output type.
+   */
+  def create[T](priorities: Seq[Int]): Graph[UniformFanInShape[T, T], NotUsed] =
```

Comment:
Shouldn't we use some more native java type here, instead of `Seq`?
I think `Array[Int]` would be rather convenient since it's easy to define such arrays, but `java.util.List[Integer]` could also work.